packages/backend/src/nest/registration/registration.service.ts

```diff
@@ -4,35 +4,75 @@ import { extractPendingCsrs, issueCertificate } from './registration.functions'
 import { ErrorCodes, ErrorMessages, PermsData, RegisterOwnerCertificatePayload, SocketActionTypes } from '@quiet/types'
 import { RegistrationEvents } from './registration.types'
 import Logger from '../common/logger'
+import { StorageService } from '../storage/storage.service'
 
 @Injectable()
 export class RegistrationService extends EventEmitter implements OnModuleInit {
   private readonly logger = Logger(RegistrationService.name)
   public certificates: string[] = []
-  private _permsData: PermsData
+  public permsData: PermsData
```

Comment:
Why is this public? If it's only for setting permsData in test, maybe let's add a setter instead? Or something simliar

Reply:
I'm happy to change it back to using a setter as it was before. It's a very minor thing. But just to explain why I changed it: I don't see much benefit of a setter/getter if it's not doing anything beyond setting/getting the value. That's what public is for and doesn't require a variable name like `_permsData`. From a caller's perspective the syntax of a setter and public are the same. If we want to avoid public variables for some reason, then I'd probably prefer using methods like `setPermsData` over JS's setter/getter functionality.